modin/test/test_partition_api.py

```diff
@@ -118,7 +142,14 @@ def test_from_partitions(axis):
             futures = [client.scatter([df1, df2], hash=False)]
         else:
             futures = client.scatter([df1, df2], hash=False)
-    actual_df = from_partitions(futures, axis)
+    actual_df = from_partitions(
+        futures,
+        axis,
+        index=index,
+        columns=columns,
+        row_lengths=row_lengths,
+        column_widths=column_widths,
```

Comment:
dtypes

Reply:
Changes related to `dtypes` were removed.